cli/scorecard/score.go

```diff
@@ -258,16 +258,29 @@ func writeResults(config *ScoringConfig, dest io.Writer, outputFormat string, ou
 
 // findViolations gets violations for the inventory and attaches them
 func (inventory *InventoryConfig) findViolations(config *ScoringConfig) error {
-	violations, err := getViolations(inventory, config)
-	if err != nil {
-		return err
-	}
+	if inventory.concurrency {
+		violations, err := getViolationsConcurrently(inventory, config)
+		if err != nil {
+			return err
+		}
 
-	err = config.attachViolations(violations)
-	if err != nil {
-		return err
+		err = config.attachViolations(violations)
```

Comment:
Since most of this code is shared between both if blocks, only the initial line (violations, err) should be gated. Though I think we should actually remove this gating entirely.